src/main/resources/application.properties

```diff
@@ -11,11 +11,13 @@ api.snyk.host=https://app.snyk.io/api/v1
 # api.snyk.disabled=true
 
 api.trustedcontent.host=https://exhort.trust.rhcloud.com/api/v1/
+api.osvnvd.host=http://localhost:8080/
```

Comment:
I should use the %dev prefix in that case as this is a requirement for testcontainers when running in dev mode (if a container is not already running). For other environments we specify the properties in the deployment templates (SRE-gitlab).
Regarding your suggestion, you can use env vars anytime to override the config properties so by doing API_OSVNVD_HOST=https://something-else would have the same effect.

Reply:
@ruromero understood..
and you're right, this is exactly the [conversion rules ](https://quarkus.io/guides/config-reference#environment-variables)
Just wanted to draw attention that if you want to customize so it will look more elegant and readable (with underscore '_' serarating between the two words , you can use instead
api.osvnvd.host=${API_OSV_NVD_HOST:http://localhost:8080/} , and you can't derive it from the conversion rules because you don't want to put property like  api.osv.nvd.host, as osv is not a parent of nvd and both are unrelated.

Reply:
Ah, well, yes we can use the hyphen and the env var will have the same name: `api.osv-nvd.host` will be `API_OSV_NVD_HOST` but I would like to find a better name for this service so, let's keep it like this as it is likely to change.
Thanks